sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -584,6 +615,11 @@ struct _pi_queue : _pi_object {
     bool InUse;
     // Record if the associated command list (if any) is a "copy" command list.
     bool IsCopyCommandList;
+    // Record the index of copy queue (in the vector of available copy queues)
+    // to which the command list (if any) will be submitted.
+    // If there is no command list, or if the command list is not a copy command
+    // list, the value is set to -1.
+    int CopyQueueIndex;
```

Comment:
can't you use CopyQueueIndex == -1 check to replace the IsCopyCommandList member?

Reply:
Yes. I think we can.